help/destinations/catalog/advertising/amazon-ads.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Amazon Ads
+description: Amazon Ads offers a range of options to help you achieve your advertising goals to registered sellers, vendors, book vendors, Kindle Direct Publishing (KDP) authors, app developers, and/or agencies. The Amazon Ads integration with Adobe Experience Platform provides turn-key integration to Amazon Ads products, including the Amazon DSP (ADSP). Using the Amazon Ads destination in Adobe Experience Platform, users are able to define advertiser audiences for targeting and activation on the Amazon DSP.  
+
+---
+
+# Amazon Ads connection {#amazon-ads}
+
+## Overview {#overview}
+
+Amazon Ads offers a range of options to help you achieve your advertising goals to registered sellers, vendors, book vendors, Kindle Direct Publishing (KDP) authors, app developers, and/or agencies.
+
+The Amazon Ads integration with Adobe Experience Platform provides turn-key integration to Amazon Ads products, including the Amazon DSP (ADSP). Using the Amazon Ads destination in Adobe Experience Platform, users are able to define advertiser audiences for targeting and activation on the Amazon DSP.  
+
+>[!IMPORTANT]
+>
+>This documentation page was created by the *Amazon Ads* team. For any inquiries or update requests, please contact them directly at *`amc-support@amazon.com`.*
+
+## Use cases {#use-cases}
+
+To help you better understand how and when you should use the *Amazon Ads* destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.
+
+### Use case #1 {#use-case-1}
```

Comment:
If this is the only use case in the doc, please:

- remove `### Use case #1 {#use-case-1}` 
- Update `Activation and Targeting Use-cases:` to `### Activation and targeting {#activation-and-targeting}`

Reply:
Done.